server/models/Interview.mjs

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const interviewSchema = new mongoose.Schema({
+  interview_id: { type: Number, required: true, unique: true },
+  user_id: { type: Number, ref: 'User', required: true },
+  application_id: { type: Number, ref: 'Job_Application', required: true },
+  interview_date: { type: Date },
+  interview_time: { type: String },
+  interview_location: { type: String },
+  interviewer_name: { type: String },
+  interview_notes: { type: String }
+});
+
+  const Interview = mongoose.model('Interview', interviewSchema);
+
+// Helper function for findOrCreateFromGoogle and findOrCreateFromGithub
```

Comment:
May want to delete or revise this comment. See also the ones for updateInterview() and deleteInterview().

Reply:
Fixed